v4/export/config.go

```diff
@@ -246,6 +250,7 @@ func (conf *Config) DefineFlags(flags *pflag.FlagSet) {
 	flags.Bool(flagTransactionalConsistency, true, "Only support transactional consistency")
 	_ = flags.MarkHidden(flagTransactionalConsistency)
 	flags.StringP(flagCompress, "c", "", "Compress output file type, support 'gzip', 'no-compression' now")
+	flags.Int(flagWriteSpeedLimit, 0, "Dump phase network speed limit. Default is 0 which means no limit. The unit is MB/s.")
```

Comment:
please make the input a string that is parsed by `ParseFileSize` (or just use `units.RAMInBytes` directly). see `flagFilesize` for reference.

Reply:
resolved.